README.md

```
@@ -140,6 +143,7 @@ You can get your API keys from:
 - Anthropic API key: https://console.anthropic.com/
 - OpenAI API key: https://platform.openai.com/api-keys
 - OpenRouter API key: https://openrouter.ai/keys
+- Gemini API key: https://aistudio.google.com/app/apikey
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)